src/internal/Subscription.ts

```diff
@@ -186,26 +136,18 @@ export class Subscription implements SubscriptionLike {
    *
    * All finalizer instances are removed to free up memory upon unsubscription.
    *
+   * TIP: In instances you're adding and removing _Subscriptions from other Subscriptions_, you should
```

Comment:
This tip is a bit hard for me to understand (most probably because I'm not native English speaker). Is there a way to rephrase it a bit?

Also, I think it has a grammatical error here: `...this a non-issue,...`.